packages/type-storage/src/fromMetadata.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017-2018 @polkadot/storage authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import Metadata from '@polkadot/api-codec/Metadata';
+
+import { createFunction } from './utils/createFunction';
+import { ModuleStorage, Storage } from './types';
+
+/**
+ * Sets the 1st letter of a string to lowercase.
+ *
+ * @param s - The string to lower first letter.
+ */
+function lowerFirstLetter (s: string) {
```

Comment:
We don't have a util/string/* section, but this would nicely fit in there. (Maybe just mark for moving). (I'm meaning in the common repo)

Reply:
OK, will put a comment and put it there on my free time.

Reply:
"free time" :)